nuxeo-features/nuxeo-platform-publisher/nuxeo-platform-publisher-core/src/main/java/org/nuxeo/ecm/platform/publisher/impl/service/PublisherServiceImpl.java

```diff
@@ -99,9 +99,9 @@
 
     @Override
     public void applicationStarted(ComponentContext context) {
-        boolean started = TransactionHelper.isTransactionActive();
+        boolean txWasStartedOutsideComponent = TransactionHelper.isTransactionActive();
```

Comment:
Unless there is a very specific need you should use `isTransactionActiveOrMarkedRollback` rather than `isTransactionActive`.